File: docs/components_page/components/badge/size.py
import dash_bootstrap_components as dbc
from dash import html

badges = html.Div(
    [
        html.H1(["Example heading", dbc.Badge("New", className="ms-1")]),
        html.H2(["Example heading", dbc.Badge("New", className="ms-1")]),
        html.H3(["Example heading", dbc.Badge("New", className="ms-1")]),
        html.H4(["Example heading", dbc.Badge("New", className="ms-1")]),
        html.H5(["Example heading", dbc.Badge("New", className="ms-1")]),
        html.H6(["Example heading", dbc.Badge("New", className="ms-1")]),
    ]
)

File: docs/components_page/components/badge/size.R
library(dashBootstrapComponents)

badges <- div(
  list(
    h1(list("Example heading", dbcBadge("New", className = "ms-1"))),
    h2(list("Example heading", dbcBadge("New", className = "ms-1"))),
    h3(list("Example heading", dbcBadge("New", className = "ms-1"))),
    h4(list("Example heading", dbcBadge("New", className = "ms-1"))),
    html$h5(list("Example heading", dbcBadge("New", className = "ms-1"))),
    html$h6(list("Example heading", dbcBadge("New", className = "ms-1")))
  )
)
